README.md

```diff
@@ -102,16 +102,23 @@ The production deployment can also be served as a hapi plugin, located in `serve
 ### Static Site Deployment with Amazon CDK
 This example was created based on the [CDK Static Site Example](https://github.com/aws-samples/aws-cdk-examples/tree/master/typescript/static-site). Below are steps to get setup. IMPORTANT: before following them, update your `cdk.json` file, replacing dummy parameters with real ones for your account and site.
 
+#### Setup your IAM user
+If you have permissions to create an IAM user and add permissions, look for and add these managed permissions:
+```
+cdk-api
+cdk-api-deploy
```

Comment:
I still need to review the updated roles, but it seems like you have this and the api playground are using the same roles.  Did you find they didn't have differences in permissions, or it wasn't worth splitting them up?

Reply:
Zoinks! This needs fixin' — these should be `cdk-strangeluv` and `cdk-strangeluv-deploy`